Metalama.Framework.DesignTime/Pipeline/DesignTimeAspectPipeline.PipelineState.cs

```diff
@@ -480,16 +480,16 @@ internal static FallibleResultWithDiagnostics<AspectPipelineConfiguration> GetCo
 
             if ( !getConfigurationResult.IsSuccessful )
             {
-                if ( state._pipeline.Logger.Error != null )
+                if ( state._pipeline.Logger.Trace != null )
                 {
                     var errors = getConfigurationResult.Diagnostics.Where( d => d.Severity == DiagnosticSeverity.Error ).ToReadOnlyList();
 
-                    state._pipeline.Logger.Error?.Log(
+                    state._pipeline.Logger.Trace?.Log(
                         $"TryGetConfiguration('{compilation.Compilation.AssemblyName}') failed: {errors.Count} errors reported." );
```

Comment:
Nit: (no need to fix this if you don't want to) The method was recently renamed from `TryGetConfiguration` to `GetConfiguration`, so the log messages in this file should be changed too.